src/locales/en/translation.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
So with `i18next`, all strings must be placed into one file? IMO, that is not as good of a developer experience as the current boilerplate's ability to define messages along side the component and then gather the strings into a single file later. It also makes it harder to break a component out into a new location/repo/package.

Reply:
You can split them up. Anyone doing slightly advanced stuff would read the docs and do it I guess. Here I wanted to give them a type-safe way to start. It could be improved in time.